### PR TITLE
Bump eks-operator to version 1.0.700

### PR DESCRIPTION
**Pull request by thedadams**
In order to fix an issue for users, we would like to do an out-of-band release of eks-operator. The image and chart are built, and merging this PR would result in a release to users.

Issue:
https://github.com/rancher/rancher/issues/31612

### CODE REVIEW

File: charts/rancher-eks-operator/1.0.700/values.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+eksOperator:
+  image:
+    repository: rancher/eks-operator
+    tag: v1.0.6
```
Comment (cbron) [inline charts/rancher-eks-operator/1.0.700/values.yaml]: Did nothing else make it into eks-operator since 2.5.6 besides this ?

Comment (thedadams) [reply charts/rancher-eks-operator/1.0.700/values.yaml]: Correct, only the fix for encoded user data made it in.

**Review by rmweir — CHANGES_REQUESTED** [review]
Can the last two commits be combined or is there a reason to keep it this way?

**Comment by thedadams** [comment]
@rmweir That was the question I had during standup today. I was wondering if that was possible because `dev-v2.5` already has the `1.0.7-rc1` commit and leaving it out of `release-v2.5` might cause conflicts later.

**Review by rmweir — APPROVED** [review]
LGTM

**Review by cbron — APPROVED** [review]

**Pull request merged**